drivers/include/can/doc.txt

```diff
@@ -0,0 +1,6 @@
+/**
+ * @defgroup    drivers_can CAN Drivers
+ * @ingroup     drivers_netdev
+ * @brief       Drivers for CAN devices
+ * @see         sys_can
```

Comment:
Can you add a `@note These devices are not covered by the @ref drivers_netdev_api`?